kalite/distributed/management/commands/setup.py

```diff
@@ -362,5 +363,11 @@ def handle(self, *args, **options):
             call_command("videoscan")
 
             # done; notify the user.
-            print("CONGRATULATIONS! You've finished setting up the KA Lite server software.")
+            print("\nCONGRATULATIONS! You've finished setting up the KA Lite server software.")
             print("You can now start KA Lite with the following command:\n\n\t%s start\n\n" % start_script_path)
+            
+            if options['interactive']:
+                if raw_input_yn("Do you wish to start the server now?"):
```

Comment:
Doesn't `kalite start` run `setup` already? We also print instructions on how to start the server after this. I feel like adding these lines of code doesn't add much convenience, except for expediting a few more keypresses.